### PR TITLE
Adjusted change reactor primitive

### PR DESCRIPTION
**Pull request by jcrugzz**
@indexzero based on our exchange in #4 i decided to just write a scenario since its easier to talk over code. It seems that the choices are sacrifice api cleanliness for multi key or sacrifice multi key. Thoughts?

### CODE REVIEW

File: lib/godot/reactor/change.js [hunk 1]
```diff
@@ -9,31 +9,28 @@ var util = require('util'),
...
-      else {
-        all.push(keys);
-      }
+  this.last;
+  this.key = key;
 
-      return all;
-    }, []);
+  // Case where you want a specified change
+  if(options) {
```
Comment (mmalecki) [inline lib/godot/reactor/change.js]: Minor style nitpicking: `if (options)`

File: lib/godot/reactor/change.js [hunk 2]
```diff
@@ -45,26 +42,29 @@ util.inherits(Change, ReadWriteStream);
...
-    var diff = true;
+      changed = false;
 
-    if (typeof self.last[key] === 'undefined'
-        || self.last[key] === data[key]) {
-      diff = false;
+  // If something changed, see if its a specified case or changed is true
+  if (typeof this.last !== 'undefined'
+      && data[this.key] !== this.last) {
+    if(this.from && this.to) {
```
Comment (mmalecki) [inline lib/godot/reactor/change.js]: Same here

**Comment by mmalecki** [comment]
LGTM. I see no problem with creating multiple reactor, this is cleaner.

**Comment by jcrugzz** [comment]
@mmalecki Word, merging.

**Pull request merged**